webservice/src/main/java/com/yahoo/covid19/models/Place.java

```diff
@@ -21,6 +22,8 @@
 public class Place {
     @Id
     private String id;
+    @Column(name = "type")
+    private String regionType;
```

Comment:
Doesn't this need to be a list of types?  See my comment about JsonType in Elide.